src/Sylius/Bundle/OrderBundle/Aggregator/AdjustmentsByLabelAggregator.php

```diff
@@ -23,12 +23,12 @@ class AdjustmentsByLabelAggregator implements AdjustmentsAggregatorInterface
      */
     public function aggregate(array $adjustments)
     {
-        $aggregatedAdjustments = array();
         foreach ($adjustments as $adjustment) {
-            if (!$adjustment instanceof AdjustmentInterface) {
-                throw new \InvalidArgumentException('Each adjustments array element must implement '.AdjustmentInterface::class.'.');
-            }
+            $this->assertElementIsAdjustment($adjustment);
```

Comment:
Yes, but then we would do some unnecessary operations on previous, good adjustments (it's not necessary to aggregate 100 adjustments if 101th is incorrect). I just didn't know which one is better option ;)